### ISSUE TITLE
[ADE LVL 1] Missing focus indicator when error triggers on required field

### ISSUE THREAD

**Issue by outerpress** [department-of-veterans-affairs/va.gov-team#111659]
> [!IMPORTANT]
> Review fix in related issue for community care scheduling: https://github.com/department-of-veterans-affairs/va.gov-team/issues/111488

## Description

When a required field isn’t filled out and I press “Continue,” it triggers an error.   

Focus moves to the error \- which is great\! \- but there’s no yellow focus indicator around the error. It’s unclear (as a visual user) that focus has moved to the error or what will happen when I press TAB.  

[Clip](https://coformaco.zoom.us/clips/share/VfNB8zDkTIK1pH1Zn2pOfw) comparing your form to the mock form patterns, which implement the focus indicator correctly  

* **Page:** [What type of care do you need](https://staging.va.gov/my-health/appointments/schedule/type-of-care), [Which VA facility would you like to go to?](https://staging.va.gov/my-health/appointments/schedule/location), [What date and time do you want for this appointment?](https://staging.va.gov/my-health/appointments/schedule/date-time), [When would you like an appointment (VA request)?](https://staging.va.gov/my-health/appointments/schedule/va-request/),  [How should we contact you (direct scheduling)?](https://staging.va.gov/my-health/appointments/schedule/contact-information), [How should we contact you (VA request)?](https://staging.va.gov/my-health/appointments/schedule/va-request/contact-information), [When would you like an appointment (community care)?](https://staging.va.gov/my-health/appointments/schedule/community-request/), [What language do you prefer?](https://staging.va.gov/my-health/appointments/schedule/community-request/preferred-language), [How should we contact you (community care)](https://staging.va.gov/my-health/appointments/schedule/community-request/contact-information)?  
* **Device:** All  
* **Orientation:** Both  
* **Expected severity level:** 1  
* **Expected experience standard:** User knows where a button or link leads  
* **How to recreate:**   
  * For example:   
    1. Go to the first page of the form  
    2. TAB to “Continue” and select it \- the error triggers, but there’s no focus indicator  
* **Recommendation:**   
  * Radio button group: This is working when you use the va-radio component. I recommend looking over that code to see what’s missing from your custom radio group implementation :)  
  * Text input: This works properly when you use va-text-input. Is there a way to use the standard VA component for Phone number and Email address?  
  * Select: This is working on the [Mock Form Patterns demo](https://staging.va.gov/mock-form-patterns/mailing-address) (try filling out everything but “Country”), draw from there?  
  * Checkbox group: Is it possible to use the VADS component for this instead of the custom component you used on “How should we contact you?”?  
  * Date pickers: There’s no existing component to draw from. Maybe there are examples in the wild of other grouped components, like va-date, that you can draw from?

---

## Development Checklist
Background: the `toggle` toggle is on (remove if not needed) 

- [ ] Task 1
- [ ] Task 2

### Design Assets
- [Figma file]() (put N/A if not needed) 

### Developer Reference

- [Swagger File]() (remove if not needed)
- [File path]() (remove if not needed)

## Definition of Done
- [ ] All tasks criteria are met
- [ ] [Technical Documentation]() is updated (must confirm if needed/or remove) 
- [ ] [Feature Reference (GitHub)](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/appointments/va-online-scheduling/feature-reference) is updated
- [ ] UX - [Feature Reference (Figma)](https://www.figma.com/design/eonNJsp57eqfPqx7ydsJY9/Feature-Reference-%7C-Appointments-FE?node-id=1150-105402&t=zRNPBqgPDVU0dDP8-1) is updated

**Comment by JunTaoLuo** [comment]
@department-of-veterans-affairs/uae-ux this is ready for validation in staging.

**Comment by Jenny-yw** [comment]
@JunTaoLuo All appts pages (direct scheduling and request) have been validated and good to go. I keep getting an error for CC referrals. Does "scheduling community care referrals" share the same flow as appts request? If so, it's good to go as well.

**Comment by JunTaoLuo** [comment]
> Does "scheduling community care referrals" share the same flow as appts request?

Unfortunately no. If there are issues with that flow, I would recommend opening a different ticket for that for the CIE team.
